contributing/index.md

```diff
@@ -2,3 +2,70 @@
+
+## Background: What's Zetkin?
+
+If you are a programmer, user interface designer, user experience designer, or have some other technical competence, you are welcome to contribute to the Zetkin project which would benefit all attached organizations.
+
+Do you want to start contributing right now?
+
+Frontend development in CSS, HTML and JavaScript (React)? You can start right away, on our Open Source projects, using GitHub Issues and Pull Requests to interact with the Zetkin maintainers.
+
+Backend development in Python or Node.js? We do this, but you need to get in touch with the Zetkin maintainers to start doing that.
+
+## Get a code collaboration account at GitHub
+
+Zetkin projects use [GitHub Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) for collaboration. You will need [a personal GitHub account](https://github.com/join).
+
+[Connecting to GitHub with SSH](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh) explains how to create and add an SSH key to your account.
+
+## Get a Slack account
+
+Ask your Zetkin maintainer for an invite to the Zetkin Slack channels, where you
+can get help from other people.
+
+## Install Docker
+
+To develop Zetkin, you need to **get a development environment started**. There
+are instructions about this in different project READMEs.
+
+All of them use
+[Docker](https://www.docker.com/), so begin by installing [Docker Desktop](https://www.docker.com/get-started) for your computer platform.
+
+### Platform installation snags
+
+Windows! To run Docker on Windows, you _may_ need to check which version of
+Windows you are running.
+
+If your Windows version is Windows Home, there is an installable piece of
+software to support Docker. [Docker documentation: Install Docker Desktop on Windows Home](https://docs.docker.com/docker-for-windows/install-windows-home/)
+
+Please ask for help from hackathon participants!
+
+## Clone a project
+
+Zetkin has three large projects you can access on [the Zetkin GitHub Organization](https://github.com/zetkin) right now:
+
+- [www](https://github.com/zetkin/www.zetk.in#getting-started) - the activist-facing website
+- [call](https://github.com/zetkin/call.zetk.in#getting-started) - a phone banking system
+- [organize](https://github.com/zetkin/organize.zetk.in#getting-started) - the organizer tool
+
+## Find a task!
+
+Zetkin has many already-described tasks to go work on, in GitHub Issues, and we labeled some of
+them as extra suitable for newcomers. [GitHub Issues labeled "Entry Level"](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Azetkin+label%3Aentry-level+no%3Aassignee).
+
+Don't be discouraged if the text makes you feel like you have no idea what is
+going on, these Issues are designed to be introduced by a Zetkin maintainer. Find someone who can explain.
+
+Also, when you find one that you like, tell the others (e.g. in the Slack
```

Comment:
I think we should mandate that contributors comment something like "I'm working on this" on the actual issue. That way, even if us maintainers forget to assign them after they let us know, other contributors will be able to tell from the issue page (comment) that someone has claimed it.